package.json

```diff
@@ -35,29 +35,45 @@
     "styleguide:analyse": "ng build styleguide --stats-json && npm run styleguide:explore",
     "styleguide:explore": "webpack-bundle-analyzer dist/styleguide/stats.json"
   },
+  "peerDependencies": {
+    "@a-ui/core": "2.1.1",
```

Comment:
do we need core and flexboxgrid as peer dependencies, or should these just be devdependencies?

Reply:
i'd leave them as peer dependencies, nobody is required to install them but this gives some indication of supported versions